sql/sql_planner.cc

```diff
@@ -118,14 +118,6 @@ class Loose_scan_opt
       but initializing them would cause a 2% CPU time loss in a 20-table plan
       search. So we initialize only if warnings would stop the build.
```

Comment:
"So we initialize only if warnings would stop the build" comment becomes outdated and needs removing